test/jdk/java/lang/invoke/indify/Indify.java

```diff
@@ -320,14 +431,11 @@ public Class<?> loadClass(String name, boolean resolve) throws ClassNotFoundExce
                         if (resolve)  resolveClass(c);
                         return c;
                     }
-                } catch (ClassNotFoundException ex) {
-                    // fall through
-                } catch (IOException ex) {
+                } catch (ClassNotFoundException | IOException ex) {
                     // fall through
                 } catch (Exception ex) {
                     // pass error from reportPatternMethods, etc.
-                    if (ex instanceof RuntimeException)  throw (RuntimeException) ex;
-                    throw new RuntimeException(ex);
+                    throw (RuntimeException) ex;
```

Comment:
Casting all remaining exceptions to `RuntimeException` ?
I would suggest to keep the original code 
or include `RuntimeException` in the fall through section above and wrap all remaining exceptions into a `RuntimeException` to replicate the original behavior.